src/Build/BackEnd/Components/ProjectCache/Experimental/ProjectCacheDescriptor.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Build.Experimental.ProjectCache
+{
+    [System.Obsolete("This class was moved to Microsoft.Build.ProjectCache namespace")]
+    public class ProjectCacheDescriptor
```

Comment:
do we need to duplicate this one? where is it used?

Reply:
let's hear from @rainersigwald for this one
my position is always to be more aggressive with deprecation :)